pos_mail_receipt/models/pos_order.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Coop IT Easy SCRLfs
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+import base64
+from odoo import fields, models, api, _
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    email_receipt_sent = fields.Boolean()
+
+    @api.model
+    def send_mail_receipt(
```

Comment:
I think that this function should be done asynchroneously, by queue job. 
don't you think ? 

like this module : https://github.com/OCA/pos/tree/12.0/pos_picking_delayed

Reply:
I'd prefer avoiding this dependency, emails are queued by default also.

I think it is better to have the e-mail sent quickly though, sometimes you need to have it to justify that you did not stole or to have a reduced parking fee.

Reply:
> I think it is better to have the e-mail sent quickly though, sometimes you need to have it to justify that you did not stole or to have a reduced parking fee.

Hi thanks for your answer. my thought was 
- about the compatibility of this module with the certification module. (that displays the hash on the bill). But I have no time to test, and the feature was introduced in V13. (I make some developmment for V12, but not yet merged).
- IFAIK, I don't think that if you use ``send_force``, the email is not queued. I think that it will wait for the sent of the mail, and I don't think that it is in the spirit of the PoS module, to wait for server response.
ref https://github.com/OCA/pos/pull/428/files#diff-cc4623d2c77df155a2db869e209c63a2R63

But well, I don't think if it's real issues, and have no time to propose a better design, so I abstain on that point, and it's not a blocking point for me. Let's go reviewing and merging this PR, and see if it's OK.

Thanks for this work !